src/components/content/heroes/aurora/weather-easter-egg/sun.tsx

```diff
@@ -0,0 +1,129 @@
+import SUN from '../../../../../assets/images/aurora/sun.png';
+import REFLECTION from '../../../../../assets/images/aurora/sun-reflection.png';
+import SUNSHINE from '../../../../../assets/images/aurora/sun-shine.png';
+import styled, { keyframes } from 'styled-components';
+import { useEffect, useState } from 'react';
+import React from 'react';
+import { getSunTime } from './weather-api';
+import { getSunlightPercentage } from './sun-percentage-calculator';
+import { Flare, FlareType } from '../flare';
+import { DefaultFlares } from '../default-flares';
+
+const rotatingAnimation = keyframes`
+    0% {
+        transform: rotate(0deg);
+    }
+    100% {
+        transform: rotate(360deg);
+    }
+`;
+
+const AuroraSunShineDiv = styled.div`
+  background-image: url(${SUNSHINE});
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: contain;
+  animation: ${rotatingAnimation} 60s linear infinite;
+  width: 200%;
+  height: 200%;
+  left: -50%;
+  top: -50%;
+  position: absolute;
+`;
+
+const AuroraSunDiv = styled.div<{ timePercent: number }>`
+  transition: all 0.5s ease-out;
+  left: calc(${(props) => props.timePercent}% - 167.5px);
+  bottom: calc(
+    ${(props) =>
+        (1 / 125) * (-(props.timePercent - 50) * (props.timePercent - 50)) +
+        60}%
+      // parabola formula to get the sun to move in a parabola
+  );
+  position: absolute;
+  background-image: url(${SUN});
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+  display: inline-block;
+  width: 335px;
+  height: 335px;
+`;
+
+const AuroraSunReflectionDiv = styled.div<{ timePercent: number }>`
+  background-image: url(${REFLECTION});
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+  display: inline-block;
+  width: 549.5px;
+  height: 539px;
+  left: calc(
+    ${(props) => props.timePercent}% +
+      ${(props) => (props.timePercent - 50) / 2}% - 274.75px
```

Comment:
`274.75px` is a super specific magic number. You got that by tinkering around?
What's the meaning? An offset? A center point? A variable name can help understanding it:

`${(props) => (props.timePercent - 50) / 2}% - ${centerX}` or similar